eslzArm/eslz-portal.json

```diff
@@ -1293,6 +1293,292 @@
                                 ]
                             }
                         },
+                        {
+                            "name": "privateDnsZones",
+                            "type": "Microsoft.Common.DropDown",
+                            "label": "Select Private DNS Zones to create",
+                            "defaultValue": "",
```

Comment:
```suggestion
```

Can we remove this so the default would be all of them selected, as per the docs state here: https://learn.microsoft.com/azure/azure-resource-manager/managed-applications/microsoft-common-dropdown#remarks

Reply:
@jtracey93 I cannot see all of the items being selected automatically even if I entirely remove "defaultValue". Am I missing something here?

Reply:
how i read it thats how it should work. Have you tested and left it for a little while and re-tested, to avoid caching issues in the portal UX

Reply:
Yeah, even if I try to test after waiting more than 45 minutes or so, still the same. It does not select them all :(